File: src/main/java/personal/ffivquizbot/event_waiter/EventWaiterProvider.java
package personal.ffivquizbot.event_waiter;

import com.jagrosh.jdautilities.commons.waiter.EventWaiter;
import lombok.Getter;
import org.springframework.stereotype.Service;

@Service
@Getter
public class EventWaiterProvider {
    private final EventWaiter eventWaiter = new EventWaiter();
}

File: src/main/kotlin/personal/ffivquizbot/eventwaiter/EventWaiterProvider.kt
package personal.ffivquizbot.eventwaiter

import com.jagrosh.jdautilities.commons.waiter.EventWaiter
import org.springframework.stereotype.Service

@Service
class EventWaiterProvider(
) {
    private val _eventWaiter = EventWaiter()

    val eventWaiter: EventWaiter
    get() = _eventWaiter
}




